UpdateService/handler.ts

```diff
@@ -90,31 +90,23 @@ export function UpdateServiceHandler(
 
     const existingService = maybeService.value;
 
-    const errorOrMaybeUpdatedService = await serviceModel.update(
-      existingService.id,
-      existingService.serviceId,
-      currentService => {
-        return {
-          ...currentService,
-          ...apiServiceToService(servicePayload),
-          serviceId
-        };
-      }
-    );
+    const errorOrUpdatedService = await serviceModel
+      .upsert({
+        ...existingService,
+        ...apiServiceToService(servicePayload),
+        kind: "INewService",
+        version: (Number(existingService.version) + 1) as NonNegativeInteger
```

Comment:
can we use `incVersion`?

Reply:
fixed in [#f21e668](https://github.com/pagopa/io-functions-admin/pull/65/commits/f21e66844748d000d7948d88490535ac7ec6ef6f)